apps/web/src/serlo-editor/plugins/fill-in-the-blanks-exercise/index.tsx

```diff
@@ -12,31 +11,40 @@ import { EditorPluginType } from '@/serlo-editor/types/editor-plugin-type'
 
 export type FillInTheBlanksMode = 'typing' | 'drag-and-drop'
 
+export type BlankId = string
+
+export type DraggableId = string
+
 export const fillInTheBlanksExercise: EditorPlugin<FillInTheBlanksExerciseState> =
   {
     Component: FillInTheBlanksExerciseEditor,
     config: {},
-    state: createFillInTheBlanksExerciseState(),
+    state: createState(),
   }
 
-export type FillInTheBlanksExerciseState = ReturnType<
-  typeof createFillInTheBlanksExerciseState
->
+export type FillInTheBlanksExerciseState = ReturnType<typeof createState>
 
-function createFillInTheBlanksExerciseState() {
+function createState() {
   const defaultMode: FillInTheBlanksMode = 'typing'
 
   return object({
     text: child({
       plugin: EditorPluginType.Text,
       config: {
         formattingOptions: [
-          ...defaultFormattingOptions,
+          TextEditorFormattingOption.code,
+          TextEditorFormattingOption.colors,
+          TextEditorFormattingOption.lists,
+          TextEditorFormattingOption.richTextBold,
```

Comment:
I understand it correctly that `math` formatting is not allowed in the first go?

Reply:
Yes! I removed it for now because the math toolbar is broken within the blanks exercise: You cannot switch from visual to latex mode.